pubspec.lock

```diff
@@ -447,4 +440,4 @@ packages:
     source: hosted
     version: "2.1.15"
 sdks:
-  dart: ">=2.3.0-dev.0.1 <3.0.0"
```

Comment:
Do you really intend to do this? It means that spread and collection if/for won't work.

Reply:
I guess not! I just used what was there. What do you suggest?

Reply:
I'd suggest just not changing this line, but @RedBrogdon would know what to do, I think.

Reply:
Unless there's a good reason not to, it'd be better to have the pubspec use sdk >=2.3.0 (>=2.3.0-dev for Flutter projects). Otherwise you can get an analyzer warning when using the UI-as-code stuff from 2.3.0.